Update cloudwatch task doc (#3206)
astronomer/docs: astro/export-cloudwatch.md

Complete the following setup to authenticate your Deployments to AWS Cloudwatch and forward your observability data to your AWS Cloudwatch instance.

:::info

> At this time, you can only export Airflow task logs from Astro to Cloudwatch. Metrics exporting is supported only on [Datadog](export-datadog.md)

Author: ```suggestion
At this time, you can only export Airflow task logs from Astro to Cloudwatch. Metrics exporting is supported only on [Datadog](export-datadog.md).
```

> At this time, you can only export Airflow task logs from Astro to Cloudwatch. If you need to export both task logs and metrics, see how to set up [forwarding data to Datadog](export-datadog.md)

Reviewer: ```suggestion
At this time, you can only export Airflow task logs from Astro to Cloudwatch. Metrics exporting is supported only on [Datadog](export-datadog.md)
```
Feel free to play with the wording here, but I wanted to remove the "If" statement as I assume users are only looking to integrate for "their" tool, and it'd be unrealistic for someone on Cloudwatch to start using Datadog for this one incompatibility.
